web/packages/teleport/src/services/discovery/types.ts

```diff
@@ -43,6 +49,30 @@ export type AwsMatcher = {
   integration: string;
   // kubeAppDiscovery specifies if Kubernetes App Discovery should be enabled for a discovered cluster.
   kubeAppDiscovery?: boolean;
+  /**
+   * InstallParams sets the join method when installing on
```

Comment:
```suggestion
   * install sets the join method when installing on
```

JSDocs above have similar minor issues where the name in the comment doesn't quite match the name in the code.